src/Monolog/Logger.php

```diff
@@ -697,4 +728,28 @@ public static function setTimezone(\DateTimeZone $tz)
     {
         self::$timezone = $tz;
     }
+
+    /**
+     * Set the minimal log level.
+     *
+     * Logger will log only records with log level >= $level.
+     *
+     * @param string $level
+     */
+    public function setLevel($level) {
+        if (!isset(static::$levelMapping[strtoupper($level)])) {
```

Comment:
In my opinion the `$levelMapping` property is redundant and we can go with `in_array(strtoupper($level), static::$levels, TRUE)`.

Reply:
Fixed.